lib/widgets/main_app_bar.dart

```diff
@@ -1,161 +1,150 @@
-                          leading: FlutterLogo(),
-                          title: Text('Four Columns'),
-                          trailing: Icon(Icons.adjust),
-                          onTap: () {/* react to the tile being tapped */}),
-                    ),
-                    FlatButton(
-                      color: Colors.purple,
-                      textColor: Colors.white,
-                      padding: EdgeInsets.all(8.0),
-                      splashColor: Colors.purple,
-                      onPressed: () {
-                        /*...*/
-                      },
-                      child: Text(
-                        "UPDATE",
-                        style: TextStyle(fontSize: 16.0),
-                      ),
-                    )
-                  ],
-                ),
-              ),
-            ),
-          ],
-        ),
-      ),
-      'custom2': CustomOverlayScreen(
-        backgroundColor: Colors.yellow[400],
-        content: Column(
-          mainAxisAlignment: MainAxisAlignment.center,
-          children: <Widget>[
-            Icon(
-              Icons.check_circle_outline,
-              color: Colors.green,
-              size: 80.0,
-            ),
-            Padding(
-                padding: EdgeInsets.only(top: 20.0),
-                child: Text("Let's start!"))
-          ],
-        ),
-      ),
-    });
-    return Container(
-      padding: const EdgeInsets.symmetric(
-        vertical: 10.0,
-        horizontal: 10.0,
-      ),
-      color: Theme.of(context).primaryColor,
-      child: Row(
-        children: [
-          Expanded(
-            child: Row(
-              mainAxisAlignment: MainAxisAlignment.spaceBetween,
-              children: [
-                // Readability Settings
-                GestureDetector(
-                  onTap: () async {
-                    OverlayScreen().show(
-                      context,
-                      identifier: 'display',
-                    );
-                  },
-                  child: Icon(
-                    Icons.apps,
-                    color: Colors.white,
-                    size: 25.0,
-                  ),
-                ),
-                // Board Name
-                Text(
-                  'Board Name',
-                  style: TextStyle(color: Colors.white, fontSize: 20.0),
-                ),
-                // Lock
-                GestureDetector(
-                  onTap: () => print('Lock'), // insert new screen here
-                  child: Icon(
-                    Icons.lock,
-                    color: Colors.white,
-                    size: 25.0,
-                  ),
-                ),
-              ],
-            ),
-          ),
-        ],
-      ),
-    );
-  }
-}
+import 'package:flutter/material.dart';
+// import 'widgets.dart';
+import 'package:overlay_screen/overlay_screen.dart';
+
+class MainAppBar extends StatelessWidget {
+  final double scrollOffset;

```

Comment:
What's the `scrollOffset` used for? I think I spotted this on a few other components, but I was just wondering!